api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryService.java

```diff
@@ -33,37 +33,39 @@ public interface ReportingQueryService {
 
   List<ReportingWorkspaceFreeTierUsage> getWorkspaceFreeTierUsage();
 
-  List<ReportingWorkspace> getWorkspaces(long limit, long offset);
+  List<ReportingWorkspace> getWorkspaceBatch(long limit, long offset);
```

Comment:
I might have gone a little overboard on the renaming, but I think these names are more explicit